### PR TITLE
[Scoper] Using different scoper.php for php 7.0

### PR DESCRIPTION
**Pull request by samsonasik**
revert original scoper.php

### CODE REVIEW

**Comment by samsonasik** [comment]
@TomasVotruba if this still make php 7.1 fail, then there is something wrong with the git add process itself, as previously, it still works on php 7.1. Let's try.

File: .github/workflows/build_scoped_rector_php70.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ name: Build Scoped Rector PHP 7.0
 on:
     push:
         branches:
-            - main
+            - never
```
Comment (samsonasik) [inline .github/workflows/build_scoped_rector_php70.yaml:8]: temporary disable scoped 7.0

**Comment by samsonasik** [comment]
the php 7.1 scoped seems working again.

**Comment by TomasVotruba** [comment]
Thanks :+1: 

https://github.com/rectorphp/rector-prefixed is passing :heavy_check_mark:

**Pull request merged**